formats/cbor/commonMain/src/kotlinx/serialization/cbor/internal/Encoding.kt

```diff
@@ -633,9 +952,55 @@ private fun Iterable<ByteArray>.flatten(): ByteArray {
 
 @OptIn(ExperimentalSerializationApi::class)
 private fun SerialDescriptor.isByteString(index: Int): Boolean {
-    return getElementAnnotations(index).find { it is ByteString } != null
+    return kotlin.runCatching { getElementAnnotations(index).find { it is ByteString } != null }.getOrDefault(false)
```

Comment:
Why `runCatching` is needed here?

Reply:
it's only a theoretical possibility, but an `IndexOutOfBoundsException` could be thrown. I'll remove it.

Reply:
well, such a call fails for the WASM target, even though we use `runCatching`. I have no explanation

Reply:
Can you elaborate on the problem? I do not see any related WASM failures

Reply:
it was an implementation bug on my end, that has ben resolved in the meantime. I removed the `runCatching` in a1a7dad6a08d0a86004bc343e546e73acddfc2ba